auth-web/src/components/pay/UnlinkedShortNameTable.vue

```diff
@@ -1,80 +1,11 @@
 <template>
   <div>
-    <ModalDialog
-      ref="accountLinkingDialog"
-      max-width="720"
-      :show-icon="false"
-      :showCloseIcon="true"
-      dialog-class="lookup-dialog"
-      :title="`Linking ${state.selectedShortName.shortName} to an Account`"
-      @close-dialog="resetAccountLinkingDialog"
-    >
-      <template #text>
-        <p
-          v-if="state.selectedAccount.accountId"
-          class="py-4 px-6 important"
-        >
-          <span class="font-weight-bold">Important:</span> Once an account is linked, all payment received
-          from the same short name will be applied to settle outstanding balances of
-          the selected account.
-        </p>
-        <h4>
-          Search by Account ID or Name to Link:
-        </h4>
-        <ShortNameLookup
-          :key="state.shortNameLookupKey"
-          @account="state.selectedAccount = $event"
-          @reset="resetAccountLinkingDialog"
-        />
-      </template>
-      <template #actions>
-        <div class="d-flex align-center justify-center w-100 h-100 ga-3">
-          <v-btn
-            large
-            outlined
-            color="outlined"
-            data-test="dialog-ok-button"
-            @click="cancelAndResetAccountLinkingDialog()"
-          >
-            Cancel
-          </v-btn>
-          <v-btn
-            large
-            color="primary"
-            data-test="dialog-ok-button"
-            @click="linkAccount()"
-          >
-            Link to an Account and Settle Payment
-          </v-btn>
-        </div>
-      </template>
-    </ModalDialog>
-    <ModalDialog
-      ref="accountLinkingErrorDialog"
-      max-width="720"
-      dialog-class="notify-dialog"
-      :title="state.accountLinkingErrorDialogTitle"
-      :text="state.accountLinkingErrorDialogText"
-    >
-      <template #icon>
-        <v-icon
-          large
-          color="error"
-        >
-          mdi-alert-circle-outline
-        </v-icon>
-      </template>
-      <template #actions>
-        <v-btn
-          large
-          color="primary"
-          data-test="dialog-ok-button"
-          @click="closeAccountAlreadyLinkedDialog"
-        >
-          Close
-        </v-btn>
-      </template>
-    </ModalDialog>
+    <ShortNameLinkingDialog
+      :isShortNameLinkingDialogOpen="state.isShortNameLinkingDialogOpen"
```

Comment:
...toRefs(state) -> means you'd only have to put in `:isShortNameLinkingDialogOpen="isShortNameLinkingDialogOpen"`